CHANGELOG.md

```diff
@@ -1,90 +1,127 @@
 # Changelog
+
 ## [Unreleased]
 
+## [1.6.1] - 2020-07-03
```

Comment:
Please put your changes in the unreleased section, I will version and date it once I publish it.

Reply:
Yep! No problem. Moved.